getting-started.md

```diff
@@ -74,6 +75,20 @@ hs.hotkey.bind({"cmd", "alt", "ctrl"}, "W", function()
 end)
 ```
 
+### <a name="spoons"></a>Spoons
+
+Spoons are pre made plugins for Hammerspoon.
+
+Have a look at the official documentation for Spoons
+[here](https://github.com/Hammerspoon/hammerspoon/blob/master/SPOONS.md)
+about how they work and how to use them.
+
+The official website with the listing of Spoons is
+[https://www.hammerspoon.org/Spoons/](https://www.hammerspoon.org/Spoons/)
+and the official repository is
+[https://github.com/Hammerspoon/hammerspoon](https://github.com/Hammerspoon/hammerspoon).
```

Comment:
Did you mean https://github.com/Hammerspoon/Spoons here?

Reply:
Ah, yes. Fixing now.

Reply:
Fixed. I'll mark this as resolved.